### DISCUSSION TITLE
NinjaMuffin.

### DISCUSSION THREAD

**Discussion by isakube** [FunkinCrew/Funkin#1821] [General]
that's not very funky fresh of you
![Screen Shot 2021-09-30 at 3 45 29 PM](https://user-images.githubusercontent.com/68293280/135539889-bf6d23fb-191c-4421-b213-ff81e7df314b.png)

**Comment by ninjamuffin99** [comment]
its a template

**Comment by catte-cloud** [comment]
what the fuck does this even mean

**Reply by isakube** [reply]
"Describe your bug in detail here" in the issue title, leading to wonderfulness such as #982
